### PR TITLE
Chapter 3: Mostly static pages

### PR DESCRIPTION
**Pull request by TuTranVan**
screenshot from 2019-02-15 09-04-35

### CODE REVIEW

**Comment by TuTranVan** [comment]
ready

**Comment by TuTranVan** [comment]
ready

**Comment by TuTranVan** [comment]
ready

**Comment by TuTranVan** [comment]
ready

**Comment by TuTranVan** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -34,18 +34,21 @@ gem 'jbuilder', '~> 2.5'
...
-  gem 'web-console', '>= 3.3.0'
+  # Access an IRB console on exception pages or by using
+  # -- <%= console %> anywhere in the code.
   gem 'listen', '>= 3.0.5', '< 3.2'
-  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
+  gem 'web-console', '>= 3.3.0'
+  # Spring speeds up development by keeping your application running
+  # -- in the background. Read more: https://github.com/rails/spring
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
```
Comment (phamvanbk11) [inline Gemfile]: dùng nháy " nhé em

File: app/views/static_pages/about.html.erb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<% provide(:title, "About") %>
+<h1><%= t "static_pages.about" %></h1>
+<p>
+  The <a href="https://www.railstutorial.org/"><em><%= t "tutorial" %></em></a>
+  is a <a href="https://www.railstutorial.org/book">book</a> and
+  <a href="http://screencasts.railstutorial.org/">screencast series</a>
+  to teach web development with
```
Comment (phamvanbk11) [inline app/views/static_pages/about.html.erb]: I18n hết nhé

**Comment by phamvanbk11** [comment]
commented

**Comment by TuTranVan** [comment]
ready

**Comment by phamvanbk11** [comment]
merged

**Pull request merged**